### PR TITLE
Fix FTBFS

### PR DESCRIPTION
**Pull request by willysr**

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libbass

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libbass

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/lua-readline

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/lua-readline

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/dropbox-python

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/dropbox-python

**Comment by willysr** [comment]
@sbo-bot: build libraries/dropbox-python

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/folly

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

File: libraries/libliftoff/libliftoff.SlackBuild [hunk 1]
```diff
@@ -68,7 +68,7 @@ rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
 rm -rf $PRGNAM-v$VERSION-$COMMIT
-tar xvf $CWD/$PRGNAM-$VERSION.tar.gz || tar xvf $CWD/$VERSION.tar.gz
+tar xvf $CWD/$PRGNAM-$VERSION.tar.gz | tar xvf $CWD/$PRGNAM-v$VERSION-$COMMIT.tar.gz
```
Comment (aclemons) [inline libraries/libliftoff/libliftoff.SlackBuild:71]: Shouldn't this be `||`

File: libraries/libliftoff/libliftoff.info [hunk 2]
```diff
@@ -2,7 +2,7 @@ PRGNAM="libliftoff"
 VERSION="0.4.1"
 HOMEPAGE="https://gitlab.freedesktop.org/emersion/libliftoff"
 DOWNLOAD="https://gitlab.freedesktop.org/emersion/libliftoff/-/archive/v0.4.1/libliftoff-0.4.1.tar.gz"
```
Comment (aclemons) [inline libraries/libliftoff/libliftoff.info:4]: You can also use the "release" link https://gitlab.freedesktop.org/emersion/libliftoff/-/releases/v0.4.1/downloads/libliftoff-0.4.1.tar.gz

That MD5SUM matches the one here already and it results in the same file name with wget or a browser (content-disposition header)

Comment (aclemons) [reply libraries/libliftoff/libliftoff.info:4]:
```sh
$ curl -L -v https://gitlab.freedesktop.org/emersion/libliftoff/-/releases/v0.4.1/downloads/libliftoff-0.4.1.tar.gz -o /dev/null 2>&1 | grep -i content-disposition
< content-disposition: attachment; filename="libliftoff-0.4.1.tar.gz"; filename*=UTF-8''libliftoff-0.4.1.tar.gz
```

File: libraries/libliftoff/libliftoff.SlackBuild (hunk 1 above)
Comment (willysr) [reply libraries/libliftoff/libliftoff.SlackBuild:71]: yeah, missed that part

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 network/argus-clients

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/libliftoff

**Comment by willysr** [comment]
@sbo-bot: build libraries/opencl-amd

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/openvdb

**Comment by willysr** [comment]
@sbo-bot: i586 libraries/openvdb

**Comment by willysr** [comment]
@sbo-bot: build i586 libraries/openvdb

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/pcl

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/plastid

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/fcitx5-chinese-addons

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/fcitx5-chinese-addons

**Comment by willysr** [comment]
@sbo-bot: build x86_64 misc/fcitx5-chinese-addons

**Comment by willysr** [comment]
@sbo-bot: build x86_64 misc/par2cmdline-tbb

**Comment by willysr** [comment]
@sbo-bot: build x86_64 misc/par2cmdline-tbb

**Comment by willysr** [comment]
@sbo-bot: build x86_64 multimedia/droidcam

**Comment by willysr** [comment]
@sbo-bot: build multimedia/droidcam

**Comment by willysr** [comment]
@sbo-bot: build x86_64 multimedia/sfnt2woff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 multimedia/sfnt2woff

**Comment by willysr** [comment]
@sbo-bot: build multimedia/sfnt2woff

**Comment by willysr** [comment]
@sbo-bot: build multimedia/sfnt2woff

**Comment by willysr** [comment]
@sbo-bot: build x86_64 multimedia/shine

**Comment by willysr** [comment]
@sbo-bot: build multimedia/shine

**Comment by willysr** [comment]
@sbo-bot: build x86_64 libraries/pcl

**Comment by aclemons** [comment]
Might be clearer if you push the things that are actually fixed and read to go to the github branch, then it won't retigger the sbolint and rev-dep checks for everything on every push since they will then already be part of the base branch?

**Pull request closed without merging**